capstone/src/main/webapp/scripts/post.js

```diff
@@ -48,22 +96,97 @@ function createPostText(post) {
   return postContent;
 }
 
+function createLikesContainer(post, likedPosts) {
+  const likesDiv = document.createElement('div');
+  likesDiv.className = "likes-div";
+
+  // Set number of likes label
+  const likesLabel = document.createElement('p');
+  likesLabel.className = "likes-label vertical-align";
+  likesLabel.id = post.postId + "likes-label";
+  const numLikes = post["likes"].length;
+  const likesString = numLikes > 1 
+    ? `${numLikes} likes` 
+    : `${numLikes} like`;
+  const labelString = numLikes == 0 
+    ? "" 
+    : likesString;
```

Comment:
If these fit in one line < 80 chars, they can be put inline. This style is mostly used to avoid going over 80 chars wide.